client/web/compose/src/components/Chart/Report/GenericChart.vue

```diff
@@ -243,6 +585,11 @@ export default {
         { value: 'center', text: this.$t('edit.yAxis.labelPosition.center') },
         { value: 'start', text: this.$t('edit.yAxis.labelPosition.bottom') },
       ],
+
+      legendType: [
+        { value: 'plain', text: 'Default' },
```

Comment:
Translations here, also make sure its scroll by default

Reply:
This is a leftover. The type is scroll by default and if `Show as list` is checked, the type will be plain